src/Uuid.php

```diff
@@ -3,9 +3,13 @@
 namespace Kblais\Uuid;
 
 use Ramsey\Uuid\Uuid as RamseyUuid;
+use Ramsey\Uuid\Exception\UnsatisfiedDependencyException;
 
 trait Uuid
 {
+    protected $uuid_version = 4;
```

Comment:
using the methods mean that user have to include the ramsi namespace, however we can remove the properties from the trait and add them to the class same as the normal trait